src/adapter.js

```diff
@@ -191,8 +191,13 @@ function KarmaReporter (tc, jasmineEnv) {
     // Any errors in top-level afterAll blocks are given here.
     handleGlobalErrors(result)
 
+    // Remove functions from called back results to avoid IPC errors in Electron
+    // https://github.com/twolfson/karma-electron/issues/47
+    let cleanedOrder = Object.assign({}, result.order);
```

Comment:
Looks like `Object.assign()` is not supported on IE11.
https://caniuse.com/#search=object.assign

I think we could loop over `Object.keys()` and copy if `typeof result[key] !== 'function'`.